arm-compute/2017-03-30/swagger/runCommands.json

```diff
@@ -0,0 +1,303 @@
+{
```

Comment:
Every swagger/configuration must have a security definitions section and it must adhere to the structure described in: https://github.com/Azure/autorest/tree/master/docs/developer/validation-rules/security-definitions-structure-validation.md